uint32 	id
uint32 	hamming_distance
float64 distance
geometry_msgs/PoseStamped stamped